### ISSUE TITLE
Patched?

### ISSUE THREAD

**Issue by eeferfef** [gato-louco/noping-account-generator#4]
E-mail or password is not correct

maybe only at me try this 
{"email":"iayfkhdpwmil@gmail.com","password":"iayfkhdpwmil"}

**Comment by gato-louco** [comment]
yeah, seems like they fixed the captcha and now they are validating it.

**Comment by gato-louco** [comment]
lmao, they definitely fixed it hahaha
even the endpoint to activate is different.
well, if someone wants to lost more time in this project, feel free to update it. for now, I recommend you use my other project, exitlag account activator
